### ISSUE TITLE
Handling inline HTML in bilara-data root: are any changes needed?

### ISSUE THREAD

**Issue by sujato** [suttacentral/bilara#99]
We recently changed bilara to show HTML as HTML in root. 

https://github.com/suttacentral/bilara/commit/f661d13483492a25ed410011d6f22d6d738de8ca

This raises the question as to how to handle such cases properly. Here's my thoughts.

- [X] HTML in root is by default rendered as HTML
    - in most cases, a user either should not add the relevant HTML to translation (as with text-critical markup) or will understand the markdown needed (as with simple bold and italic).
- [ ] user can toggle HTML as plain text?
     - in cases where a translator wants to see the HTML, let them toggle it? Or is this not necessary? After all, a user can always check this in the browser inspector anyway. is it worth adding complication to the app for what I hope is a marginal use? Probably not!
- [ ] add "copy-to-translate" button to root field. 
     - This will simply copy the root verbatim to the translate field. This would be handy for HTML as well as spelling names in Pali and Sanskrit. But again, it really just replicates browser functionality: just double-click the root field, copy and paste.

**Comment by sabbamitta** [comment]
I don't think it is as easy as this. Not in the case of Sutta translation, there it is not so complicated. But when translating the essays that belong to the "site" section, it is difficult. There are many links in these essays, and the link HTML is much more complex as just the link element and the URL. I don't know how I could get to see these links. I need to have them for the translation, however. Most of the essays start already with a "table of contents" which has links to the different paragraphs of the essay, which I need to see in order to add them to the translation. To my understanding, a way to see the exact HTML is essential. (Until this is done, I stop translating the remaining "site" essays for now.)

**Comment by sujato** [comment]
Ok, well maybe we should go ahead and add the toggle-to-see-html button. We can implement it as a switch on the toolbar, turn it on and the HTML appears in the root, just as it was before.

**Comment by sabbamitta** [comment]
Thank you, that would be great.
